cli/operator/node.go

```diff
@@ -608,12 +609,16 @@ func setupSSVNetwork(logger *zap.Logger) (networkconfig.NetworkConfig, error) {
 		if len(byts) != 4 {
 			return networkconfig.NetworkConfig{}, errors.New("custom domain type must be 4 bytes")
 		}
+		// Assign domain type as-is for Genesis.
 		networkConfig.GenesisDomainType = spectypes.DomainType(byts)
-		if networkConfig.PastAlanFork() {
-			logger.Info("custom domain type is ineffective now after Alan fork", fields.Domain(networkConfig.GenesisDomainType))
-		} else {
-			logger.Info("running with custom domain type", fields.Domain(networkConfig.GenesisDomainType))
-		}
+		// Assign domain type incremented by 1 for Alan.
+		alanDomainType := binary.LittleEndian.Uint32(byts) + 1
```

Comment:
Why do you prefer this solution over having both `CustomGenesisDomainType` and `CustomAlanDomainType` in config?

Reply:
@nkryuchkov custom domain type is a temporary feature we released to reduce load on mainnet, and it will be removed as soon as the Alan fork happens.

plus, there are only 2 custom domains deployed right now on mainnet and we're aware of them so we know the +1 can't cause conflicts into the fork :)

Reply:
@y0sher @moshe-blox alright, can we add a comment for this in the description of the `CustomDomainType` config option?

Reply:
@nkryuchkov added comment in help msg and also changed to BigEndian to increment the least significant byte (the right-most) like we did manually in `mainnet.go`